src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.html

```diff
@@ -35,10 +36,11 @@
              let-column="column"
              let-value="value"
              let-row="row">
-  <i *ngIf="row.iconClass"
+  <i *ngIf="row.remainingDays < this.expirationWarningAlert"
      i18n-title
      title="User's password is about to expire"
-     class="{{ icons.warning }}"
-     [ngClass]="row.iconClass"></i>
+    [class.icon-danger-color]="row.remainingDays < this.expirationDangerAlert"
+    [class.icon-warning-color]="row.remainingDays < this.expirationWarningAlert && row.remainingDays >= this.expirationDangerAlert"
```

Comment:
@epuertat I feel like a pipe here would make it more complex than needed since here it needs a color and in the other template it needs a border-color. Let me know what you think.

Reply:
Agree! Pipes should also bring some reusability, and not just that ad-hoc solution.